winconf.pri

```diff
@@ -53,7 +53,7 @@ CONFIG(debug, debug|release) {
 # Enable backtrace support
 CONFIG += strace_win
 
-win32-g++ {
+*-g++* {
```

Comment:
It is not documented. According to [SO answers](http://stackoverflow.com/questions/14523148), it will.
I don't know who to get or set this string in qmake.